content/departments/cloud/technical-docs/v2.0/onprem_data_migration.md

```diff
@@ -0,0 +1,249 @@
+# On-prem data migration to Cloud v2
+
+This process describes the current state of how to do a full data migration of an on-prem instance to a [Cloud v2](./index.md) instance.
+On-prem-to-Cloud data migrations are currently owned by [Implementation Engineering](../../../technical-success/ie/index.md), but the process is documented in Cloud as it pertains to Cloud infrastructure.
+
+> WARNING: This process is still a work-in-progress. For more details, see [RFC 760](https://docs.google.com/document/d/1IAgXmv2TbtU_rWXtph-KFc3qbqswREIAxO1rGALuoMQ/edit#) and the [tracking issue](https://github.com/sourcegraph/customer/issues/1525), or cc @bobheadxi (Robert Lin).
+> This page was adapted only on a best-effort basis for Cloud v2 from [the legacy MI v1.1 page](../v1.1/index.md) - it has currently not yet been thoroughly tested for Cloud v2.
+
+## Requirements
+
+To qualify for a data migration, the customer must:
+
+- have a Sourcegraph instance on v3.20.0 or later
+  - note: where possible, strongly encourage the customer to upgrade to their on-prem instance to the latest version of Sourcegraph first.
+- use databases on Postgres 11 or later
```

Comment:
We don't support 11 https://docs.sourcegraph.com/admin/postgres#version-requirements

Reply:
At some point we did support 11, so it's likely there are 3.20-ish instances that are on it. IIRC it was the upgrade _to_ 11 that caused issues, hence the requirement here - happy to bump that though

Reply:
Bumped the requirement regardless 👍